### PR TITLE
1.1.3

### PR DESCRIPTION
**Pull request by abique**

### CODE REVIEW

**Comment by abique** [comment]
Generated pkgconfig file looks good:
```
prefix=/usr/local
includedir=${prefix}/include

Name: clap
Description: The interface headers for the CLAP audio plugin API
Version: 1.1.3

Cflags: -I${includedir}
```

File: include/clap/entry.h [hunk 1]
```diff
@@ -16,8 +16,8 @@ extern "C" {
 //   - /usr/lib/clap
 //
 // Windows
-//   - %CommonFilesFolder%/CLAP/
-//   - %LOCALAPPDATA%/Programs/Common/CLAP/
+//   - %COMMONPROGRAMFILES%\CLAP
+//   - %LOCALAPPDATA%\Programs\Common\CLAP
```
Comment (robbert-vdh) [inline include/clap/entry.h:20]: Should this change be mentioned in the changelog? Since the earlier version was apparently incorrect.

Comment (abique) [reply include/clap/entry.h:20]: Yes, I'll do it.

**Review by defiantnerd — APPROVED** [review]
Approved by me..

**Pull request merged**